src/Sarif.Driver/Sdk/SdkResources.resx

```diff
@@ -203,4 +203,11 @@
   <data name="ERR0997_NoValidAnalysisTargets" xml:space="preserve">
     <value>No valid analysis targets were specified.</value>
   </data>
+  <data name="ERR0997_ExceptionAccessingFile" xml:space="preserve">
+    <value>An exception was raised accessing a file specified on the command-line ('{0}'). Exception information:
```

Comment:
`command-line` as a noun isn't hyphentated. (I'm not wild about hyphenating it as an adjective either, _e.g._, "command-line options", but that's moot.)